app_dart/lib/src/service/scheduler.dart

```diff
@@ -406,9 +406,15 @@ class Scheduler {
     late CiYaml ciYaml;
     log.info('Attempting to read presubmit targets from ci.yaml for ${pullRequest.number}');
     if (commit.branch == Config.defaultBranch(commit.slug)) {
+      // This fails when we attempt the second call to getCiYaml since the retry
+      // options did not set a high enough delay factor.
       final Commit totCommit = await generateTotCommit(slug: commit.slug, branch: Config.defaultBranch(commit.slug));
       final CiYaml totYaml = await getCiYaml(totCommit);
-      ciYaml = await getCiYaml(commit, totCiYaml: totYaml);
+      // TODO consider adding this as default to all calls to getCiYaml.
```

Comment:
```dart
// TODO(ricardoamador): Add this as default to all calls to getCiYaml. https://github.com/flutter/flutter/issue/#####
```

https://dart-lang.github.io/linter/lints/flutter_style_todos.html

Reply:
Okay I need to bookmark and read that this weekend! Thank you.

Reply:
Removed.